### ISSUE TITLE
Share button makes rapid inputs when held

### ISSUE THREAD

**Issue by celerizer** [lakka-switch/Lakka-LibreELEC#4]
Hotkey inputs when using the "Enable hotkeys" function (Settings > Input > Input Hotkey Binds > Enable Hotkeys) with the Share button trigger every frame; Pressing Share button + R for fast forward toggle for example will quickly toggle fast forward on and off, meaning it takes a bit of luck to make it stay toggled correctly.

All other buttons seem to work, this is also with the default behavior of "Share button = screenshot" disabled. Git version 2a2e732a79.

**Comment by natinusala** [comment]
Do other buttons work for the same function ?

**Comment by celerizer** [comment]
Yes, I've tried most other buttons with the railed Joy-Cons (D-pad, clicked sticks, Home button, triggers) and the Share button is the only one that acts like this.

**Comment by celerizer** [comment]
Actually, it looks like this happens for any input; mapping RetroPad A to Share button and holding it will cause Mario to do small jumps repeatedly.

**Comment by natinusala** [comment]
So it comes from the Screenshot button ?

**Comment by celerizer** [comment]
That's what it looks like, yeah.

**Comment by natinusala** [comment]
Then it's an issue with the Joy-Con rail driver, let me look into it
